pkg/cmd/upgrade/clustermanager/exec.go

```diff
@@ -107,8 +107,8 @@ func (o *Options) run() error {
 	}
 	output = append(output, out...)
 
-	if o.wait && !o.ClusteradmFlags.DryRun {
```

Comment:
This makes sense to me, we wait only if the wait flag is true.

Reply:
actually this is bug fixed before, to avoid [Race Condition](https://github.com/open-cluster-management-io/clusteradm/issues/47)